admin/grants.sql

```diff
@@ -227,10 +229,12 @@ TO ROLE MASKING_ADMIN;
 -- Allow for data engineer role to create tasks
 -- https://docs.snowflake.com/en/user-guide/tasks-intro#creating-tasks
 USE ROLE ACCOUNTADMIN;
-GRANT EXECUTE MANAGED TASK ON ACCOUNT TO ROLE DATA_ENGINEER; --noqa: PRS
+GRANT EXECUTE TASK, EXECUTE MANAGED TASK ON ACCOUNT TO ROLE taskadmin;
 GRANT CREATE DATABASE ON ACCOUNT TO ROLE DATA_ENGINEER; --noqa: PRS
+USE ROLE securityadmin;
+GRANT taskadmin TO ROLE DATA_ENGINEER; --noqa: PRS
+GRANT taskadmin TO ROLE SYSADMIN; --noqa: PRS
```

Comment:
why didn't `sysadmin` need this before? The privileges with `taskadmin`?

Reply:
Good question - I'm just taking this opportunity to also allow SYSADMIN to manage tasks. Prior to this, I would have to assume ACCOUNTADMIN to do so.